backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/ResetPasswordQueryService.kt

```diff
@@ -0,0 +1,22 @@
+package app.ehrenamtskarte.backend.auth.webservice.schema
+
+import app.ehrenamtskarte.backend.auth.database.AdministratorEntity
+import app.ehrenamtskarte.backend.auth.database.Administrators
+import app.ehrenamtskarte.backend.auth.webservice.schema.types.Administrator
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.time.LocalDateTime
+
+@Suppress("unused")
+class ResetPasswordQueryService {
+    @GraphQLDescription("Verify password reset link")
+    fun checkPasswordResetLink(resetKey: String): Administrator {
```

Comment:
We should also have a `project` argument as input, and make sure the admin is in the correct project.